common/install.sh

```diff
@@ -67,42 +70,91 @@ ignorewarning() {
     ui_print "   Exiting..."
     abort
   fi
-}
+fi
 
-if $MAGISK; then
-  magiskpolicy --live "create system_server sdcardfs file" "allow system_server sdcardfs file { write }"
+ui_print " "
+ui_print "   Removing remnants from past Pix3lify installs..."
+# remove /data/resource-cache/overlays.list
+OVERLAY='/data/resource-cache/overlays.list'
+if [ -f "$OVERLAY" ]; then
+  ui_print "   Removing $OVERLAY"
+  rm -f "$OVERLAY"
 fi
 
-if [ "$PX1" ] || [ "$PX1XL" ] || [ "$PX2" ] || [ "$PX2XL" ] || [ "$PX3" ] || [ "$PX3XL" ] || [ "$N5X" ] || [ "$N6P" ]; then
-  ui_print " "
-  ui_print "   Pix3lify is only for non-Google devices!"
-  ignorewarning
+if [ "$SLIM" == false -a "$FULL" == false -a "$OVER" == false -a "$BOOT" ] then
+  if keytest; then
+    FUNCTION=choose
+  else
+    FUNCTION=chooseold
+    ui_print "   ! Legacy device detected! Using old keycheck method"
+    ui_print " "
+    ui_print " - Vol Key Programming -"
+    ui_print "   Press Vol Up:"
+    $FUNCTION "UP"
+    ui_print "   Press Vol Down:"
+    $FUNCTION "DOWN"
+  fi
+
+  if ! $SLIM && ! $FULL & ! $OVER ! $BOOT; then
+    ui_print " "
+    ui_print " - Slim Options -"
+    ui_print "   Do you want to enable slim mode (heavily reduced featureset, see README)?"
+    ui_print "   Vol Up = Yes, Vol Down = No"
+    if $FUNCTION; then
+      SLIM=true
+    else
+      FULL=true
+    fi
+    if $FULL; then
+      ui_print " "
+      ui_print " - Overlay Options -"
+      ui_print "   Do you want the Pixel accent or overlay features enabled?"
+      ui_print "   Vol Up = Yes, Vol Down = No"
+      if $FUNCTION; then
+        OVER=true
+      fi 
+    fi
+    ui_print " "
+    ui_print " - Animation Options -"
+    ui_print "   Do you want the Pixel boot animation?"
+    ui_print "   Vol Up = Yes, Vol Down = No"
+    if $FUNCTION; then
+      BOOT=true
+    fi
+  else
+    ui_print " Options Specified in Zipname!"
+  fi
 fi
 
-if [ "$OOS" ]; then
+#had to break up volume options this way for basename zip for users without working vol keys
+if $SLIM; then 
   ui_print " "
-  ui_print "   Pix3lify has been known to not work and cause issues on devices running OxygenOS!"
-  ignorewarning
+  ui_print "   Enabling slim mode..."
+  rm -f $INSTALLER/common/PhenotypePrefs.xml
```

Comment:
We don't need to delete any content of the `common` folder

Reply:
Didn't even see that. @ThatNerdyPikachu had this from the original profiles branch